napari/components/viewer_model.py

```diff
@@ -402,12 +409,22 @@ def reset_view(self) -> None:
         # zoom is definied as the number of canvas pixels per world pixel
         # The default value used below will zoom such that the whole field
         # of view will occupy 95% of the canvas on the most filled axis
+
+        if 0 <= margin < 1:
+            scale_factor = 1 - margin
+        else:
+            raise ValueError(
+                trans._(
+                    f'margin must be between 0 and 1; got {margin} instead.',
+                    deferred=True,
+                )
```

Comment:
Does `trans` support f-string now?

Reply:
@jni What with this?

Reply:
I don't know, but I don't think it's worth holding up the PR for. Translations are not currently used and won't be for a long time. There's a chance this whole branch won't exist by the time we actually need to use translations in production.

Reply:
@Czaki fixed with f5fe4f4fa